README.md

```diff
@@ -431,6 +431,8 @@ ANSWERS.addComponent('SearchBar', {
   clearButton: true,
   // Optional, redirect search query to url
   redirectUrl: 'path/to/url',
+  // Optional, target frame for the redirect url, defaults to current frame
+  redirectUrlTarget: '_self',
```

Comment:
Did Rose say anything about grouping these two config options? Something like:

```
redirect: {
  url: '...',
  target: '...'
}
```
Personally, I prefer related options grouped like this. It's not a big deal, but I think it allows people to compartmentalize. It would mean that the `redirectUrl` is now deprecated, which might not be worth it.

Reply:
Yeah, that was actually an alternative considered in the item! This is what that portion of the description said:

> Alternatives considered:
> 
> Nest it a new object, but this just adds more mess for backwards compatibility
> 
> redirect {
>    url: test.answers.com
>    target: "top"
> }
> 
>

Reply:
Okay! Sounds like we decided it wasn't worth the compatibility headache.